.storybook/main.js

```diff
@@ -22,17 +20,17 @@ module.exports = {
           {
             test: /\.mjs$/,
             include: /node_modules/,
-            type: "javascript/auto",
+            type: 'javascript/auto',
           },
         ],
       },
       resolve: {
         ...resolve,
-        plugins: [
-          ...(resolve.plugins || []),
-          new TsconfigPathsPlugin()
-        ]
-      }
-    };
-  }
-}
\ No newline at end of file
+        plugins: [...(resolve.plugins || []), new TsconfigPathsPlugin()],
+      },
+    }
+  },
+  features: {
+    emotionAlias: false,
```

Comment:
👍

Reply:
Haha yeah it took a long time to find that!